website/content/docs/security/acl/acl-roles.mdx

```diff
@@ -250,6 +286,35 @@ service_prefix "" {
 }
 ```
 
+</Tab>
+
+<Tab heading="ENT">
+
+```hcl
+partition "<partition>"
+  namespace "default" {
+    # Allow the agent to register its own node in the Catalog and update its network coordinates
```

Comment:
I really don't think we should put documentation inside code comments. We should place the description in front of the example to introduce the code behavior.